packages/contracts-bedrock/src/dispute/OutputBisectionGame.sol

```diff
@@ -0,0 +1,551 @@
+        if (_claimIndex == 0) {
+            subgameAtRootResolved = true;
+        }
+    }
+
+    /// @inheritdoc IDisputeGame
+    function rootClaim() public pure returns (Claim rootClaim_) {
+        rootClaim_ = Claim.wrap(_getArgFixedBytes(0x00));
+    }
+
+    /// @inheritdoc IDisputeGame
+    function extraData() public pure returns (bytes memory extraData_) {
+        // The extra data starts at the second word within the cwia calldata and
+        // is 32 bytes long.
+        extraData_ = _getArgDynBytes(0x20, 0x20);
+    }
+
+    /// @inheritdoc IDisputeGame
+    function gameData() external view returns (GameType gameType_, Claim rootClaim_, bytes memory extraData_) {
+        gameType_ = gameType();
+        rootClaim_ = rootClaim();
+        extraData_ = extraData();
+    }
+
+    ////////////////////////////////////////////////////////////////
+    //                       MISC EXTERNAL                        //
+    ////////////////////////////////////////////////////////////////
+
+    /// @inheritdoc IInitializable
+    function initialize() external {
+        // SAFETY: Any revert in this function will bubble up to the DisputeGameFactory and
+        // prevent the game from being created.
+        //
+        // Implicit assumptions:
+        // - The `gameStatus` state variable defaults to 0, which is `GameStatus.IN_PROGRESS`
+
+        // Set the game's starting timestamp
+        createdAt = Timestamp.wrap(uint64(block.timestamp));
+
+        // Set the root claim
+        claimData.push(
+            ClaimData({
+                parentIndex: type(uint32).max,
+                claim: rootClaim(),
+                position: ROOT_POSITION,
+                clock: LibClock.wrap(Duration.wrap(0), Timestamp.wrap(uint64(block.timestamp))),
+                countered: false
+            })
+        );
+
+        // Persist the L1 head hash of the parent block.
+        // TODO(clabby): There may be a bug here - Do we just allow the dispute game to be invalid? We can
+        //               always just create another, but it is possible to create a game where the data was not
+        //               already available on L1.
+        l1Head = Hash.wrap(blockhash(block.number - 1));
+    }
+
+    /// @notice Returns the length of the `claimData` array.
+    function claimDataLen() external view returns (uint256 len_) {
+        len_ = claimData.length;
+    }
+
+    ////////////////////////////////////////////////////////////////
+    //                          HELPERS                           //
+    ////////////////////////////////////////////////////////////////
+
+    /// @notice Verifies the integrity of an execution bisection subgame's root claim. Reverts if the claim
+    ///         is invalid.
+    /// @param _rootClaim The root claim of the execution bisection subgame.
+    function verifyExecBisectionRoot(Claim _rootClaim, uint256 /* _parentIndex */ ) internal pure {
```

Comment:
This seems to just allow the constructor args to bork the game from progressing?